osquery/events/events.cpp

```diff
@@ -687,8 +687,8 @@ Status EventFactory::registerEventPublisher(const PluginRef& pub) {
     specialized_pub->state(EventState::EVENT_SETUP);
     if (!status.ok()) {
       // Only start event loop if setUp succeeds.
-      LOG(INFO) << "Event publisher failed setup: " << type_id << ": "
-                << status.what();
+      LOG(WARNING) << "Event publisher failed setup: " << type_id << ": "
```

Comment:
These are usually innocuous and already glen too much attention for newcomers, I'd suggest keeping them `INFO`.